consts.php

```diff
@@ -107,12 +107,131 @@
     ],
 ];
 
+// use hardcoded.php to bulk update update this after creating a .cow.pat.json
+// for multiple versions, use an array e.g. silverstripe-mymodule => ['2.3', '2.4']
 const INSTALLER_TO_REPO_MINOR_VERSIONS = [
     '4.10' => [
-        'silverstripe-html5' => '2.3',
+        'agency-extensions' => '2.6',
+        'comment-notifications' => '2.2',
+        'cwp' => '2.9',
+        'cwp-core' => '2.9',
```

Comment:
And 2.10? Or is 2.10 a 4.11 branch? Either way it's missing.

Reply:
4.11, have updated